src/core/opamProcess.ml

```diff
@@ -212,6 +212,12 @@ let default_verbose () = OpamCoreConfig.(!r.verbose_level) >= 2
 let is_verbose_command c =
   OpamStd.Option.default (default_verbose ()) c.cmd_verbose
 
+let command_cmd c = c.cmd
+let command_args c = c.args
+let command_dir c = c.cmd_dir
+let command_env c = c.cmd_env
+let command_allow_stdin c = c.cmd_stdin
```

Comment:
Please use same naming than existing
```suggestion
let cmd_of_command c = c.cmd
let args_of_command c = c.args
let dir_of_command c = c.cmd_dir
let env_of_command c = c.cmd_env
let allow_stdin_of_command c = c.cmd_stdin
```

Reply:
I'm not sure about using this naming convention for those accessors. The `*_of_*` functions in OCaml usually imply some kind of processing (e.g. conversion from one type to the other)

Accessors are more straightforward. Furthermore I think it isn't clear for users what those do without reading the implementation, especially for `cmd_of_command` it isn't clear to me from the name what the difference between `cmd_of_command` and `string_of_command` is. `command_cmd` looks more like the typical conversion for accessors to me.

Reply:
I would generally agree, which is why originally I had them using the more usual accessor "convention". In the end I am ok with whatever OPAM maintainers prefer.

Reply:
Good question. The idea behind my proposal is to keep it homogeneous with existing ones.  We'll end up with `text_of_command` and `command_args`, both direct accessors. 
`string_of_` is another naming convention, in opam it is always used to have the full string of a type (with Module.to_string)

Reply:
The only existing one is `text_of_command`, the rest are not accessors. With that in mind i think it's fine to have two naming conventions.

Reply:
I've pushed a separate commit renaming them, it can be removed if desired before merging.